### ISSUE TITLE
YoloV8-pretrained Crowdhuman detector

### ISSUE THREAD

**Issue by SuperbTUM** [mikel-brostrom/yolo_tracking#909]
### Search before asking

- [X] I have searched the Yolov8 Tracking [issues](https://github.com/mikel-brostrom/yolov8_tracking/issues) and found no similar enhancement requests.

### Description

Hello, thanks for your hard work! I have a quick suggestion for you as I saw someone train a [yolov8-based human detector](https://github.com/truong11062002/yolov8_for_human_detection). The followings are test results based on that repo and you can get access to the checkpoints in either pytorch or onnx version from the download page. Should be stronger than current yolov5m model, and lighter at the same time. Thanks.

<html>
<body>

P | R | mAP50 | mAP50-95
-- | -- | -- | --
0.857 | 0.743 | 0.847 | 0.598

</body>
</html>

### Use case

You may replace the current yolov5 detector with the yolov8 detector as I suggested above. It should lead to better re-identification and tracking accuracy & performance on various datasets.

**Comment by mikel-brostrom** [comment]
Thanks! I have been waiting for a Yolov8 CrowdHuman model for quite a long time. Seems that this S version is better than the Yolov5m version that I have been using previously: https://github.com/deepakcrk/yolov5-crowdhuman/issues/24. Will definitely check this out later today.

**Comment by homoludens** [comment]
For my dataset deepakcrk/yolov5-crowdhuman works much better than mentioned  truong11062002/yolov8_for_human_detection, but I am not managing to run deepakcrk/yolov5-crowdhuman with your project, and judging by comments in some issues it looks like it is possible.


Do you have any suggestion how to use crowdhuman_yolov5m.pt weights?

**Comment by mikel-brostrom** [comment]
> For my dataset deepakcrk/yolov5-crowdhuman works much better than mentioned truong11062002/yolov8_for_human_detection

Yup, `deepakcrk/yolov5-crowdhuman` works much better for `MOT16/17` than `truong11062002/yolov8_for_human_detection`

**Comment by mikel-brostrom** [comment]
> Do you have any suggestion how to use crowdhuman_yolov5m.pt weights?

Easiest way would to use an older release @homoludens

**Comment by SuperbTUM** [comment]
@mikel-brostrom Did you validate the model accuracy on crowdhuman? Is it because of model capacity and generalization ability? Maybe it is always best to train a large model.

**Comment by mikel-brostrom** [comment]
There are may different sets of bboxes when training on crowdhuman: head bounding-box, human visible-region bounding-box, human full-body bounding-box and crowd. Maybe I am missing more... I have no idea about the details in this particular training, but for some reason it is much worse. Yolov5m has very similar COCO performance to Yolov8s. So I would expect Yolov8s crowdhuman to give similar results to Yolov5m crowdhuman. But this is not the case and I have not enough information to draw deeper conclusions.

**Comment by homoludens** [comment]
Thanks, I managed to use v8.0 release and works nicely. Thank you for your work. 


I have same experience with `truong11062002/yolov8_for_human_detection`  version on my recording, I need only head count since it is a large crowd, and `deepakcrk/yolov5-crowdhuman` works great for that, even `person` class is much better. Maybe we should ask @deepakcrk to train v8 version so it would work with your project.

**Comment by mikel-brostrom** [comment]
>  and deepakcrk/yolov5-crowdhuman works great for that, even person class is much better

Yup, agree @homoludens 

> Maybe we should ask @deepakcrk to train v8 version so it would work with your project.

That would be great. But I remember asking and he did not have access to the code anymore. If I remember correctly

**Comment by SuperbTUM** [comment]
I just figured out that this repo does not use CrowdHuman for training. Instead, it used a small-scale custom dataset from a single camera. That should be the reason why it performs worse than expected. Sorry for the confusion.

**Comment by mikel-brostrom** [comment]
Ok, then I am closing this :smile:. Thanks for pointing this out @SuperbTUM
